training/training.py

```diff
@@ -1,8 +1,9 @@
 from keras.models import *
 from keras.layers import *
+import pandas as pd
+import numpy as np
```

Comment:
Minor comment: In PEP8, import order is first grouped by the package visibility, and then by alphabetical order. For example:

```
import abc
import os

import numpy as np
import pandas as pd

from . import my_local_module
import other_local_module
```